### DISCUSSION TITLE
4/28(금) View 객체를 조사해주세요

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#113] [General]
오늘은 View에 대한 것을 알아보아야 봐주시고 todoList 다음주까지 todolist 만들어 주세요 그리고 먹젠 코드로 실전 연습한번 해볼 것인데 그래서 개발을 조금 일찍 시작할 것 같습니다 😆

**Comment by Eunho0922** [comment]
네
